jdaviz/utils.py

```diff
@@ -249,6 +249,10 @@ def is_not_wcs_only(layer):
     return not is_wcs_only(layer)
 
 
+def layer_is_not_dq(data):
+    return not data.label.endswith('[DQ]')
```

Comment:
The surest way to identify a DQ layer is to rely on the data label if we're always producing it consistently. I think we can rely on this for now as written, but we could instead, e.g., add a private metadata flag.

Reply:
In #2718 there is a `app._jdaviz_helper._loaded_mask_cube` variable that we can use instead of this. That way, if the DQ data is renamed, the `_loaded_mask_cube` should be able to track that (it does not currently do that in that PR but it would be a useful addition). We've been moving away from using data labels since we are planning (at some point in the future) to enable label renaming. I'm also worried about the scenario where a user manually loads the DQ layer with a custom name, which would mean it is not caught by this line.